index-aqua.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: subsite-galaxy
+website: https://aqua.usegalaxy.eu
+subdomain: aqua
+---
+
+# Welcome to Aqua's Galaxy
+
+![aqua](https://github.com/MarkusKonk/website/assets/2863409/8e5afe32-11aa-4611-8170-65b5e4503bae)
```

Comment:
At the moment I do not know how this kind of link is generated, but the fact that it points to your personal fork means that it will break if you ever decide to delete your fork.

I think you should upload the picture to the repository (if not uploaded already) and use a _relative_ link (I hope it works). Please provide information on the license of this picture so that it can be ensured that we are using it lawfully.

In addition, I see that you bothered to provide an [alt text](https://www.w3schools.com/tags/att_img_alt.asp). Given that you did the effort, I'd like to suggest you push this further and use something more meaningful than "aqua", it will make blind people happier.

Reply:
Hi @kysrpex,
Alright, thanks. For now, I will remove the picture since we anyway need to revise the entire styling. I wasn't even sure if the picture will stay there. So I will postpone this issue ;).